qrisk/stats.py

```diff
@@ -41,17 +38,23 @@
     MONTHLY: MONTHS_PER_YEAR
 }
 
-def cum_returns(returns, starting_value=None):
+
+def cum_returns(returns, starting_value=0):
     """
     Compute cumulative returns from simple returns.
 
     Parameters
     ----------
     returns : pd.Series
-        Daily returns of the strategy, noncumulative.
-         - See full explanation in tears.create_full_tear_sheet.
+        Returns of the strategy as a percentage, noncumulative.
+         - Time series with decimal returns.
+         - Example:
+            2015-07-16    -0.012143
+            2015-07-17    0.045350
+            2015-07-20    0.030957
+            2015-07-21    0.004902.
     starting_value : float, optional
-       The starting returns (default 1).
+       The starting returns (default is 0).
```

Comment:
We don't need to specify the default in the docs if the default value is present in the signature